include/aspect/boundary_temperature/dynamic_core.h

```diff
@@ -92,7 +92,7 @@ namespace aspect
      * shell geometry in which the temperature at the outer surfaces are constant
      * and the core-mantle boundaries (CMB) temperature is calculated by core energy balance.
      * The formulation of core energy balance are from:
-     * \cite{NPB+04} The influence of potassium on core and geodynamo evolution.
+     * \cite NPB+04 The influence of potassium on core and geodynamo evolution.
```

Comment:
Wait, does doxygen actually do this right in the HTML output? How cool -- I didn't realize that's possible!